### PR TITLE
Adding Employment Functionality

### PR DESCRIPTION
**Pull request by ArijElfaki**

### CODE REVIEW

File: src/actions/index.js [hunk 1]
```diff
@@ -172,3 +172,16 @@ export function RSVP(RSVPRecord, history) {
...
 }
+
+// Add employment history
+export function addEmployment(personID,companyID,startDate,endDate, history) {
+    return (dispatch) => {
+        axios.post(`${ROOT_URL}/users/employment/add`, personID)
+            .catch((error) => {
+                return dispatch(authError(error.response));
+            }).then((response) => {
+                history.push('/employment');
```
Comment (seandartmouth) [inline src/actions/index.js:183]: I don't see an '/employment' page added, did you plan on doing that in a future PR?

Comment (seandartmouth) [inline src/actions/index.js:183]: I don't see an '/employment' page added, did you plan on doing that in a future PR?

File: src/actions/index.js [hunk 2]
```diff
@@ -172,3 +172,16 @@ export function RSVP(RSVPRecord, history) {
             });
     };
 }
+
+// Add employment history
+export function addEmployment(personID,companyID,startDate,endDate, history) {
+    return (dispatch) => {
+        axios.post(`${ROOT_URL}/users/employment/add`, personID)
```
Comment (seandartmouth) [inline src/actions/index.js:179]: Did you add the /users/employment/add endpoint to the backend? I didn't see a PR there

File: src/components/employmentlist.js [hunk 3]
```diff
@@ -43,6 +45,7 @@ class EmploymentList extends Component {
                 <div>
                     {this.getEmploymentList()}
                 </div>
+                <Button onClick={() => this.props.history.push('/event')}>Back</Button>
```
Comment (ArijElfaki) [inline src/components/employmentlist.js:48]: going to change it to /employment forgot to change from event lol

File: src/components/addEmployment.js [hunk 4]
```diff
@@ -119,7 +119,7 @@ class addEmploy extends Component {
                             </Form.Group>
                         </Form.Row>
                     </Form>
-
+                    <Button onClick={() => this.props.history.push('/event')}>Back</Button>
```
Comment (ArijElfaki) [inline src/components/addEmployment.js:122]: going to change it to /employment forgot to change from event lol

File: src/components/employmentlist.js [hunk 5]
```diff
@@ -43,6 +45,7 @@ class EmploymentList extends Component {
...
             </div>
         );
     }
```
Comment (ArijElfaki) [inline src/components/employmentlist.js:51]: +export default withRouter(connect(null, { fetchEmployment })(EmploymentList));

File: src/actions/index.js [hunk 6]
```diff
@@ -172,3 +172,44 @@ export function RSVP(RSVPRecord, history) {
...
+                    callback(response.data.data);
+                }
+            });
+    };
+}
+
+// Post new event then return to events page
+export function deleteEmployment(personID, companyID, history) {
+    return (dispatch) => {
+        axios.delete(`${ROOT_URL}//deleteEmployment/${personID}/${companyID}`)
```
Comment (seandartmouth) [inline src/actions/index.js:206]: Are there supposed to be two /'s before deleteEmployment?

File: src/components/employment.js [hunk 7]
```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+// eslint-disable-next-line no-unused-vars
```
Comment (seandartmouth) [inline src/components/employment.js:3]: I think you can delete this comment.

File: src/components/employment.js [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+        super(props);
+
+        this.state = {};
+    }
+
+    render() {
+        return (
+            <div className="new-content">
+                <Error />
+                <NavLink to="/listemployment" exact>My Employment History</NavLink>
```
Comment (seandartmouth) [inline src/components/employment.js:19]: isn't if employmentlist not listemployment?

File: src/components/employmentlist.js [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+class EmploymentList extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            employmentList: [],
+        };
+    }
+
+    componentDidMount() {
+        this.props.fetchEmployment(this.props.user.id, (employmentList) => { this.setState({ employmentList }); });
```
Comment (a-matusewicz) [inline src/components/employmentlist.js:18]: user id is undefined

**Pull request merged**